src/jit/codegenxarch.cpp

```diff
@@ -5868,7 +5885,22 @@ void CodeGen::genCallInstruction(GenTreePtr node)
         else
 #endif // _TARGET_X86_
         {
-            regNumber returnReg = (varTypeIsFloating(returnType) ? REG_FLOATRET : REG_INTRET);
+            regNumber returnReg;
+            // TODO-Cleanup: For UNIX AMD64, we should not be allocating a return register for struct
```

Comment:
Opened #2551 - I don't think it is critical, but there is some unnecessary code being generated.